packages/pageflow/src/editor/api/index.js

```diff
@@ -71,15 +71,38 @@ export const EditorApi = Object.extend(
     /**
      * List of available file import plugins
      * @alias fileImporters
-     *  
      */
     this.fileImporters = new FileImporters();
   },
 
+  /**
+   * Configure editor for entry type.
+   *
+   * @param {string} name
```

Comment:
currently the `name` doesn't seem to go anywhere?

Reply:
Yes. Since there is only ever one entry type defined in the editor js at the moment, we don't need the name currently. I left it in to align the API with the other register methods and also in case we need it down the line.